src/hooks/AllPaymentHooks.res

```diff
@@ -418,6 +421,92 @@ let useRedirectHook = () => {
             },
           )
         }
+      | "third_party_sdk_session_token" => {
```

Comment:
can we please abstract out all this logic i.e. logic to trigger plaid and do a retrieve post completion of this pis flow to some other file, AllPayment hooks should be concise and readable.

Reply:
moved most of the code to `PlaidHelperHook.res` file. 👍